package.json

```diff
@@ -34,11 +34,11 @@
     "release": "npm t && standard-version && npm publish"
   },
   "dependencies": {
-    "got": "^11.8.3",
     "semver": "^7.1.1",
     "yargs": "^16.2.0"
   },
   "devDependencies": {
+    "@types/node": "^20.14.11",
```

Comment:
There was in implicit dependency on these types which was satisfied by bringing in got. Why it had it as a prod dep I don't think I understand, but removing `got` broke the type generation.

Reply:
it should be node 18, given engines.node, no?